src/System.Runtime.Serialization.Formatters/tests/TypeSerializableValue.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+
+namespace System.Runtime.Serialization.Formatters.Tests
+{
+    public class TypeSerializableValue
+    {
+        public TypeSerializableValue(string base64Blob, TargetFrameworkMoniker platform)
+        {
+            Base64Blob = base64Blob;
+            Platform = platform;
+        }
+
+        public string Base64Blob { get; }
+
+        public TargetFrameworkMoniker Platform { get; }
+
+        public static int GetPlatformIndex(TypeSerializableValue[] blobs)
+        {
+            var blobList = blobs.ToList();
+
+            // .NET Framework
+            if (PlatformDetection.IsFullFramework)
+            {
+                if (!PlatformDetection.IsNetfx471OrNewer)
```

Comment:
If I'm running on <= net470 I will never get an index back for blobs that haven't changed with net471. Why did you swap the condition?